.github/workflows/notebook-digest-updater.yaml

```diff
@@ -90,7 +89,15 @@ jobs:
                 echo $output
                 sed -i "s|${image}=.*|${image}=$output|" manifests/base/params.env
               done
-              git fetch origin ${{ env.DIGEST_UPDATER_BRANCH }} && git pull origin ${{ env.DIGEST_UPDATER_BRANCH }} && git add manifests/base/params.env && git commit -m "Update images for release N via ${{ env.DIGEST_UPDATER_BRANCH }} GitHub action" && git push origin ${{ env.DIGEST_UPDATER_BRANCH }}
+              if [[ $(git status --porcelain | wc -l) -gt 0 ]]; then
```

Comment:
Totally agree with this fix! I saw `--porcelain` option for this once my fix PR got merged! :P 
Thanks!